### ISSUE TITLE
Translations: Browser support for date and time formatting

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1622]

**Comment by colinmurphy01** [comment]
Might be possible that Drupal can handle this?

**Comment by colinmurphy01** [comment]
Two part:
- Drupal part
- Browsers and operating systems

**Comment by eric-gade** [comment]
@colinmurphy01 I think this directly relates to https://github.com/weather-gov/weather.gov/issues/1623 and should be conducted as part of that work. @sspj-does-weather what do you think?

**Comment by sspj-does-weather** [comment]
I agree. We should consolidate all the date-related translation tickets and do them in conjunction with #2170 .
